hudi-common/src/main/java/org/apache/hudi/common/table/cdc/HoodieCDCUtils.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.common.table.cdc;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+
+import org.apache.hudi.exception.HoodieException;
+
+public class HoodieCDCUtils {
+
+  public static final String CDC_LOGFILE_SUFFIX = "-cdc";
+
+  /* the `op` column represents how a record is changed. */
+  public static final String CDC_OPERATION_TYPE = "op";
+
+  /* the `ts_ms` column represents when a record is changed. */
+  public static final String CDC_COMMIT_TIMESTAMP = "ts_ms";
```

Comment:
i think `ts` would be sufficient. this timestamp is based on timeline instant time, right? which may involve beyond ms granularity so we may want to be flexible here with the key

Reply:
ok.

Reply:
see https://github.com/apache/hudi/pull/6476#discussion_r974703425
so, we will choose `ts` or `ts_ms`. @xushiyan @alexeykudinkin

Reply:
@xushiyan since we chose to stay compatible w/ Debezium let's stick w/ `ts_ms` (i left similar comment regarding `ts`)